cmd/mimir/config-descriptor.json

```diff
@@ -1924,7 +1924,7 @@
           "kind": "field",
           "name": "enable_promql_engine_fallback",
           "required": false,
-          "desc": "If set to true and the streaming engine is in use, fall back to using the Prometheus PromQL engine for any queries not supported by the streaming engine.",
+          "desc": "If set to true and the Mimir PromQL engine is in use, fall back to using the Prometheus PromQL engine for any queries not supported by the Mimir PromQL engine.",
```

Comment:
I thought we were calling it "Mimir Query Engine"? It leaves open the possibility of extending it and we've been using the MQE acronym.

Reply:
I was debating this - the flag name uses the term "PromQL engine", so I was trying to be consistent with that.

I'll change the term here.